dcos/emitting.py

```diff
@@ -163,7 +163,7 @@ def _page(output, pager_command=None):
     output = six.text_type(output)
 
     if pager_command is None:
-        pager_command = 'less -R'
+        pager_command = 'less'
```

Comment:
I think we did this because we wanted to handle color tty encoding. Did we test this manually?